src/lava/magma/core/model/py/ports.py

```diff
@@ -48,6 +48,13 @@ def peek(self):
         pass
 
     def probe(self) -> bool:
+        """Executes probe method of all csp ports and accumulates the returned
+        bool values with and operation. The accumulator acc is initialized to
```

Comment:
Capitalize "and" into "AND" to distinguish it from the regular word.

Reply:
Resolved!